django_netjsonconfig/widgets.py

```diff
@@ -27,8 +27,8 @@ def media(self):
     def render(self, name, value, attrs={}):
         attrs['class'] = 'vLargeTextField jsoneditor-raw'
         html = """
-<a href="#advanced_editor" class="button json-editor-btn-edit normal-mode">{0}</a>
```

Comment:
Is this button needed at all?

Since it is hidden I think it may be not needed anymore, what do you think? In order to remove it you will have to move the logic of returning to normal mode elsewhere.